src/parser.pegjs

```diff
@@ -1,7 +1,114 @@
 {
+  var parse = require('./parse');
+  var utils = require('./utils');
+  var conj = utils.conj;
 }
 
 
-start = dialogue
+start
+  = ws* dialogue:dialogue ws*
+  { return dialogue }
 
-dialogue = .*
+
+dialogue 'dialogue'
+  = title:dialogueTitle ws* properties:properties? ws* sequences:sequences?
+  {
+    return conj(properties || {}, {
+      title: title,
+      sequences: sequences || []
```

Comment:
Does this mean we can't have properties with the keys `title` or `sequences`? If so, maybe `properties` should be a sub-object of dialogue instead of having the keys places directly in the dialogue?